api/wikis/serializers.py

```diff
@@ -7,8 +7,27 @@
 
 from framework.auth.core import Auth
 
+
 class WikiSerializer(JSONAPISerializer):
 
+    def to_representation(self, data, envelope='data'):
+        if data.node.is_registration:
+            serializer = RegistrationWikiSerializer(data, context=self.context)
```

Comment:
Similar to the last "make registrations behave properly" PR you did, is this going to just serve up the information on a /nodes/ link even if it's a registration? I think last time we had to have a base view/serializer and then derive the node and registrations versions from there.